account_invoice_report_grouped_by_picking/views/report_invoice.xml

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template
-        id="report_invoice_document"
-        inherit_id="account.report_invoice_document"
-        priority="99"
-    >
-        <xpath expr="//t[contains(@t-foreach, 'lines')]/t[3]" position="replace">
-
-    </xpath>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//t[contains(@t-foreach, 'lines')]/t[3]" position="attributes">
+            <attribute name="invisible">True</attribute>
```

Comment:
Is this working in qweb reports? I think you have to work with `display: None`, or put a `t-if="False"` clause for avoiding its visualization.